Project.toml

```diff
@@ -46,7 +46,7 @@ BifurcationKit = "0.3"
 CairoMakie = "0.12"
 Combinatorics = "1.0.2"
 DataStructures = "0.18"
-DiffEqBase = "6.83.0"
+DiffEqBase = "< 6.159.0"
```

Comment:
We aren't releasing this anytime soon given all the current open issues, but I'd like to have CI run and docs build so we can keep merging stuff...

Reply:
The problem is that you need OrdinaryDiffEqCore >= 1.10.0 in order for DiffEqBase >=6.159.0 to work properly. What's not obvious to me is why CI is pulling in OrdinaryDiffEqCore 1.9.0 instead of 1.10.0. Currently working on fixing this....

Reply:
Ah, okay, it's caused by the compat bound on SciMLBase:
```
ERROR: Unsatisfiable requirements detected for package SciMLBase [0bca4576]:
 SciMLBase [0bca4576] log:
 ├─possible versions are: 1.0.0 - 2.60.0 or uninstalled
 ├─restricted to versions 0.0.0 - 2.57.1 by Catalyst [479239e8], leaving only versions: 1.0.0 - 2.57.1
 │ └─Catalyst [479239e8] log:
 │   ├─possible versions are: 14.4.1 or uninstalled
 │   └─Catalyst [479239e8] is fixed to version 14.4.1
 ├─restricted by compatibility requirements with JumpProcesses [ccbc3e58] to versions: 2.46.0 - 2.60.0, leaving only versions: 2.46.0 - 2.57.1
 │ └─JumpProcesses [ccbc3e58] log:
 │   ├─possible versions are: 9.0.0 - 9.14.0 or uninstalled
 │   └─restricted to versions 9.13.2 - 9 by Catalyst [479239e8], leaving only versions: 9.13.2 - 9.14.0
 │     └─Catalyst [479239e8] log: see above
 └─restricted by compatibility requirements with OrdinaryDiffEqCore [bbf590c4] to versions: 2.57.2 - 2.60.0 — no versions left
   └─OrdinaryDiffEqCore [bbf590c4] log:
     ├─possible versions are: 1.0.0 - 1.10.1 or uninstalled
     └─restricted to versions 1.10.0 by an explicit requirement, leaving only versions: 1.10.0
```
I'll need to figure out what the right compat bounds are to make sure that Catalyst et all pulls in the right versions of OrdinaryDiffEqCore for the DiffEqBase version in use. Apologies, I should have figured out the compat story as part of my PR.

Reply:
Ok, thanks for investigating. We should probably just have a DiffEqBase dep here anyways, which I’ve now added, and we can cap that for now. If/when we are ready to uncap SciMLBase we can uncap DiffEqBase and hopefully this will all work.